integration_tests/add-custom-barbell-exercise.integration.test.tsx

```diff
@@ -108,3 +116,107 @@ describe("User Journey: Add Custom Barbell Exercises", () => {
     // but I can't get _anything_ to work, try this again sometime.
   });
 });
+
+describe("User Journey: Can use components to edit from the default values", () => {
```

Comment:
This is where the new test actually is. Main thing going on here is I render the page, click the "add" button to show the form, then click basically every control that exists inside of it and then add the exercise to the db and assert that it has values that make sense based on what I clicked on.